app/models/mixins/container_archived_mixin.rb

```diff
@@ -0,0 +1,14 @@
+module ContainerArchivedMixin
+  def archived?
+    ems_id.nil?
+  end
+
+  # Needed for metrics
+  def my_zone
+    if ext_management_system.present?
+      ext_management_system.my_zone
+    elsif old_ext_management_system.present?
```

Comment:
`old_ext_management_system` seems great, but why does `my_zone` reference the old ems?